{
  "uid" : "ec7a726ad9e892f4",
  "name" : "An uncaught error was detected outside of a test",
  "fullName" : "An uncaught error was detected outside of a test",
  "historyId" : "c59643030e1e776644fd5d1d6aa79cfc",
  "time" : {
    "start" : 1644858914297,
    "stop" : 1644858914766,
    "duration" : 469
  },
  "status" : "failed",
  "statusMessage" : "The following error originated from your test code, not from Cypress.\n\n  > Cannot read properties of undefined (reading 'signals')\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.",
  "statusTrace" : "TypeError: The following error originated from your test code, not from Cypress.\n\n  > Cannot read properties of undefined (reading 'signals')\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at normalizeSignal (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:93338:1)\n    at Array.map (<anonymous>)\n    at getSignals (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:93318:53)\n    at getSignalsByName (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:93225:38)\n    at Object.653../realtime.js (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:93239:21)\n    at o (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:1:265)\n    at eval (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:1:316)\n    at Object.546.human-signals (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:74684:25)\n    at o (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:1:265)\n    at eval (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:1:316)",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 2,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "The following error originated from your test code, not from Cypress.\n\n  > Cannot read properties of undefined (reading 'signals')\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.",
    "statusTrace" : "TypeError: The following error originated from your test code, not from Cypress.\n\n  > Cannot read properties of undefined (reading 'signals')\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at normalizeSignal (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:93338:1)\n    at Array.map (<anonymous>)\n    at getSignals (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:93318:53)\n    at getSignalsByName (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:93225:38)\n    at Object.653../realtime.js (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:93239:21)\n    at o (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:1:265)\n    at eval (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:1:316)\n    at Object.546.human-signals (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:74684:25)\n    at o (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:1:265)\n    at eval (https://react-redux.realworld.io/__cypress/tests?p=cypress\\integration\\UI\\Test.feature:1:316)",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "38aee298a99c2298",
      "name" : "UI/Test.feature:2022-02-14T17:15:14.329Z",
      "source" : "38aee298a99c2298.png",
      "type" : "image/png",
      "size" : 112055
    }, {
      "uid" : "39b4f198a5f05708",
      "name" : "video recording",
      "source" : "39b4f198a5f05708.mp4",
      "type" : "video/mp4",
      "size" : 2060888
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "62f773eff28f7411",
      "status" : "failed",
      "statusDetails" : "The following error originated from your test code, not from Cypress.\n\n  > Cannot read properties of undefined (reading 'signals')\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.",
      "time" : {
        "start" : 1644858796092,
        "stop" : 1644858796589,
        "duration" : 497
      }
    }, {
      "uid" : "e713d571f49133df",
      "status" : "failed",
      "statusDetails" : "The following error originated from your test code, not from Cypress.\n\n  > Cannot read properties of undefined (reading 'signals')\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.",
      "time" : {
        "start" : 1644850901026,
        "stop" : 1644850904624,
        "duration" : 3598
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "ec7a726ad9e892f4.json",
  "parameterValues" : [ ]
}